jbi/actions/steps.py

```diff
@@ -35,12 +35,20 @@ def create_issue(context: ActionContext, **parameters):
     """Create the Jira issue with the first comment as the description."""
     bug = context.bug
 
+    # If not specified, issue type will be either 'Bug' or 'Task'
+    issue_type_map = parameters.get(
+        "issue_type_map", {"enhancement": "Task", "task": "Task", "defect": "Bug"}
+    )
+    if not isinstance(issue_type_map, dict):
+        raise TypeError("The 'issue_type_map' parameter must be a dictionary.")
+    issue_type = issue_type_map.get(bug.type, "Task")
+
     # In the payload of a bug creation, the `comment` field is `null`.
     # We fetch the list of comments to use the first one as the Jira issue description.
     comment_list = bugzilla.get_client().get_comments(bug.id)
     description = comment_list[0].text if comment_list else ""
 
-    jira_create_response = jira.create_jira_issue(context, description)
+    jira_create_response = jira.create_jira_issue(context, description, issue_type)
```

Comment:
Maybe something to track in a follow-on, but should we have a "pre-flight" check here to ensure that either the configured issue or default that we pick is available as an issue type on the project? This PR allows configuration, but it's possible that even the custom issue type mapping could drift from what's available in Jira.

Reply:
https://github.com/mozilla/jira-bugzilla-integration/issues/575